### DISCUSSION TITLE
Slow work of scripts from examples.

### DISCUSSION THREAD

**Discussion by Cyberaxe77** [ria-com/nomeroff-net#219] [Q&A]
Dear developers! Thank you for such an amazing library.
Now the question...
It takes 15 to 20 seconds from the start of any script from the examples to license plate recognition. I suspect that I am doing something wrong. Please tell me where is my mistake?
System Arch linux x86_64, 10th generation i3 processor and 8 GB RAM, python 3.10, all dependencies are installed as specified by you on github.
Thank you in advance.
P.S. Sorry for bad english. Google translate...

**Comment by ApelSYN** [comment]
When the module is launched, a lot of routine and preparatory operations go through before the model starts working, the most expensive operation is loading all the necessary models.

The strategy is as follows: you design the application as a service, which, for example, via the HTTP protocol, starts receiving requests from your application and calls the number detection function and returns the result in the response, in which case the detection will be much faster and more optimal.

We asked for examples of such an application for different frameworks, you can see the code at the link:
https://github.com/ria-com/nomeroff-net/tree/master/examples/py/rest_examples

**Reply by Cyberaxe77** [reply]
Thank you very much for such a prompt response.
I understood your idea. Thanks for the help.
